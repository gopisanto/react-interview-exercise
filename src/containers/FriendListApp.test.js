import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';

import FriendListApp from './FriendListApp';
import {store} from './App';

it('<FriendListApp /> should render', () => {
    const wrapper = renderer.create(<Provider store={store}><FriendListApp /></Provider>).toJSON();

    expect(wrapper).toMatchSnapshot();
});