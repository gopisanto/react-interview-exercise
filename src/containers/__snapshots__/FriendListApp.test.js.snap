exports[`test <FriendListApp /> should render 1`] = `
<div
  className={undefined}>
  <h1>
    The FriendList
  </h1>
  <div
    onKeyDown={[Function]}>
    <input
      autoFocus="true"
      className="form-control"
      onChange={[Function]}
      placeholder="Type the name of a friend"
      type="text"
      value="" />
    <div
      className={undefined}>
      <label
        className={undefined}>
        Gender:
      </label>
      <label
        className="radio-inline">
        <input
          checked={false}
          name="gender"
          onClick={[Function]}
          type="radio"
          value="male" />
        Male
      </label>
      <label
        className="radio-inline">
        <input
          checked={false}
          name="gender"
          onClick={[Function]}
          type="radio"
          value="female" />
        Female
      </label>
    </div>
  </div>
  <div>
    <ul
      className={undefined}>
      <li
        className={undefined}>
        <div
          className={undefined}>
          <i
            aria-hidden="true"
            className="fa fa-2x fa-female" />
          <div
            className={undefined}>
            <div>
              <span>
                Theodore Roosevelt
              </span>
            </div>
            <div>
              <small>
                xx friends in common
              </small>
            </div>
          </div>
        </div>
        <div
          className={undefined}>
          <button
            className="btn btn-default undefined"
            onClick={[Function]}>
            <i
              className="fa fa-star" />
          </button>
          <button
            className="btn btn-default undefined"
            onClick={[Function]}>
            <i
              className="fa fa-trash" />
          </button>
        </div>
      </li>
      <li
        className={undefined}>
        <div
          className={undefined}>
          <i
            aria-hidden="true"
            className="fa fa-2x fa-male" />
          <div
            className={undefined}>
            <div>
              <span>
                Abraham Lincoln
              </span>
            </div>
            <div>
              <small>
                xx friends in common
              </small>
            </div>
          </div>
        </div>
        <div
          className={undefined}>
          <button
            className="btn btn-default undefined"
            onClick={[Function]}>
            <i
              className="fa fa-star-o" />
          </button>
          <button
            className="btn btn-default undefined"
            onClick={[Function]}>
            <i
              className="fa fa-trash" />
          </button>
        </div>
      </li>
    </ul>
    <div>
      <div
        className={undefined}>
        <span
          className="undefined"
          disabled={true}
          onClick={[Function]}>
          <<
        </span>
        <span
          className="undefined"
          disabled={true}
          onClick={[Function]}>
          <
        </span>
        <span
          className="undefined"
          onClick={[Function]}>
          1
        </span>
        <span
          className=""
          onClick={[Function]}>
          2
        </span>
        <span
          className=""
          disabled={false}
          onClick={[Function]}>
          >
        </span>
        <span
          className=""
          disabled={false}
          onClick={[Function]}>
          >>
        </span>
        <span
          className={undefined}>
          1 of 2
        </span>
      </div>
    </div>
  </div>
</div>
`;
